warehouse/core/src/test/java/datawave/ingest/mapreduce/job/SplittableRFileInputFormatTest.java

```diff
@@ -0,0 +1,197 @@
+package datawave.ingest.mapreduce.job;
+
+import static org.apache.accumulo.core.conf.Property.TABLE_CRYPTO_PREFIX;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.AbstractMap;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.accumulo.core.conf.DefaultConfiguration;
+import org.apache.accumulo.core.crypto.CryptoFactoryLoader;
+import org.apache.accumulo.core.data.Key;
+import org.apache.accumulo.core.data.Value;
+import org.apache.accumulo.core.file.FileSKVIterator;
+import org.apache.accumulo.core.file.FileSKVWriter;
+import org.apache.accumulo.core.file.rfile.RFileOperations;
+import org.apache.accumulo.core.spi.crypto.CryptoEnvironment;
+import org.apache.accumulo.core.spi.crypto.CryptoService;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.mapreduce.InputSplit;
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class SplittableRFileInputFormatTest {
+    private Configuration config;
+
+    @Before
+    public void setup() {
+        Logger.getRootLogger().setLevel(Level.WARN);
+        config = new Configuration();
+    }
+
+    @After
+    public void cleanup() {
+
```

Comment:
If you made your `tmpFile` a class level variable, you could do something like:
```
if (tmpFile != null && tmpFile.exists())
  tmpFile.delete()
```
That'd get rid of the try/finally in all of your tests.

Reply:
will update